RELEASE.md

```diff
@@ -1,3 +1,29 @@
+# Release 2.2.1
+
+## Bug fixes and minor changes
+- Fix response code for vector store timeout, change it from 429 to 504 (https://github.com/marqo-ai/marqo/pull/763)
+
+# Release 2.2.0
+
+## New features
+* Support filtering on document ID with structured indexes. This was already supported with unstructured indexes ([#749](https://github.com/marqo-ai/marqo/pull/749))
+* New structured index data types: `long`, `double`, `array<long>` and `array<double>` for a higher precision and range of values Available for indexes created with Marqo 2.2+ ([#722](https://github.com/marqo-ai/marqo/pull/722))
+* Higher precision numeric fields for unstructured indexes. Unstructured indexes created with Marqo 2.2+ will use double precision floats and longs for a higher precision and wider range of values ([#722](https://github.com/marqo-ai/marqo/pull/722))
```

Comment:
Maybe `use double precision floats and longs` -> `use double precision floats and long precision integers`  ?

Reply:
Unfortunately I can't rephrase things as these notes have been published already. I just had forgotten to add them here in the repo